src/command_modules/azure-cli-interactive/HISTORY.rst

```diff
@@ -14,11 +14,8 @@ Release History
 * Fixes Interactive errors with the profile 2017-03-09-profile-preview (#3587)
 * Allows '--version' as a parameter for interactive mode (#3645)
 * Stop Interactive Mode from Throwing errors from Validation completions (#3570)
-* Fix various pylint disable rules
 * Progress Reporting for template deployments (#3510)
-* Fix method-hidden pylint disable rule
 * Improves the muting of parse args + appends history with gestures (#3434)
```

Comment:
I have no idea what this means.  Is it customer facing?  If so we should reword it or remove it.